About.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About | WORLD CONFERENCE ALERTS 2022, 2023</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="home.js" defer></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div id="portfolio" class="wrapper">
+    <header class="nav-container">
+      <div class="top-hader">
+        <nav>
+        <ul class="uls">
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/facebook(1).png" alt="facebook logo"></a></li>
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+           </li>
+          <li class="glow-on-hover">English</li>
+          <li class="glow-on-hover">MyPage</li>
+          <li class="glow-on-hover">Layout</li>
+        </ul>
+       </nav>
+       </div>
+       <div class="header2">
+        
+        <nav class="nav">
+           <div class="logoabut">
+            <a href="index.html"> <img class="logo-img1" src="images/logo.png" alt="logo cc Global summit 2015" width="100" height="50"></a>
+             </div>
+             <figure class="humburger">
+              <span  class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </figure>
+             <ul class="nav-items" id="nav-items">
+              <li><a href="" class="closebtn">x</a></li> 
+              <li><a href="index.html" class="nav-item ">Home</a> </li>
+              <li><a href="About.html" class="nav-item nav-actives">About</a> </li>
+              <li><a href="#program" class="nav-item">Program</a> </li>
+             <li><a href="#locate" class="nav-item">Sponsor</a> </li>
+             <li><a href="#locate" class="nav-item CcChampaing">CC Champaing</a> </li>
+            </ul>
+        </nav>
+      </div>
+    </header>
+  <main class="cultural-days-wrap" id="main">
+    <section class="about new">
+      <h1 class="about-tite2">"hello! sharing world</h1>
+      <h1 class="about-titel">WORLDWIDE EVENT PROMOTER</h1>
```

Comment:
-[ ] Kindly, as a good practice 🍰, you should have only one `<h1>` tag, please remove one and change it for another semantic tag. 📟